### PR TITLE
changes to make apple-silicon M1 / M2 local build possible

### PR DESCRIPTION
**Pull request by DerThorsten**

### CODE REVIEW

File: recipes/recipes/cross-python_emscripten-32/recipe.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ package:
 #   - path: _sysconfigdata__emscripten_.py
 
 build:
-  number: 7
+  number: 12
```
Comment (wolfv) [inline recipes/recipes/cross-python_emscripten-32/recipe.yaml:13]: maybe increase only 1 version?

File: testing/package_testing.py [hunk 2]
```diff
@@ -33,7 +37,7 @@ def has_pytest_files(recipe_dir):
 def create_test_env(pkg_name, prefix):
     # cmd = ['$MAMBA_EXE' ,'create','--prefix', prefix,'--platform=emscripten-32'] + [pkg_name] #+ ['--dryrun']
     cmd = [
-        f"""$MAMBA_EXE create --yes --prefix {prefix} --platform=emscripten-32   python "pyjs>=0.18.0" pytest numpy {pkg_name}"""
+        f"""$MAMBA_EXE create --yes --prefix {prefix} --platform=emscripten-32   python numpy "pyjs>=0.18.0" pytest  {pkg_name}"""
```
Comment (wolfv) [inline testing/package_testing.py:40]: does this matter?

File: testing/package_testing.py [hunk 3]
```diff
@@ -57,20 +61,37 @@ def get_node_binary():
...
     old_cwd = os.getcwd()
 
     if has_pytest_files(recipe_dir):
-
         pkg_name = recipe["package"]["name"]
 
-        with tempfile.TemporaryDirectory() as temp_dir:
-
+        with temp_work_dir(work_dir) as temp_dir:
+            # temp_dir = "/Users/thorstenbeier/debug"
```
Comment (wolfv) [inline testing/package_testing.py:92]: remove?

File: testing/package_testing.py [hunk 4]
```diff
@@ -57,20 +61,37 @@ def get_node_binary():
...
             prefix = os.path.join(temp_dir, "prefix")
+            print("prefix", prefix)
```
Comment (wolfv) [inline testing/package_testing.py:94]: remove?

**Review by wolfv — CHANGES_REQUESTED** [review]
nice, exciting that this is working!

File: testing/package_testing.py (hunk 2 above)
Comment (DerThorsten) [reply testing/package_testing.py:40]: nah .. just debugging..removing and re-adding

File: testing/package_testing.py (hunk 3 above)
Comment (DerThorsten) [reply testing/package_testing.py:92]: jop

File: testing/package_testing.py (hunk 4 above)
Comment (DerThorsten) [reply testing/package_testing.py:94]: jop

**Review by wolfv — APPROVED** [review]

**Pull request merged**